ocfp/ocfp.yml

```diff
@@ -5,24 +5,33 @@ exodus:
   service_broker_domain:     (( concat "autoscalerservicebroker." params.cf_system_domain ))
 
 instance_groups:
```

Comment:
Can we use `migrate_from` to keep data?  Otherwise, we may lose all the existing configurations

Reply:
I'm open to suggestions

Reply:
It works as it is, let's move forward and circle back with a point release if necessary.